cmd/artifact.go

```diff
@@ -34,6 +40,48 @@ func NewArtifactCmd(ctx context.Context, opt *commonoptions.CommonOptions) *cobr
 		DisableFlagsInUseLine: true,
 		Short:                 "Interact with Falco artifacts",
 		Long:                  "Interact with Falco artifacts",
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
Most of the content of this function should be factored out. Just leaving a note here. We can do that later.